vmc/resource_vmc_public_ip_test.go

```diff
@@ -43,6 +50,8 @@ func testAccCheckVmcPublicIpExists(name string, publicIpResource *model.PublicIp
 		}
 		uuid := rs.Primary.Attributes["id"]
 		displayName := rs.Primary.Attributes["display_name"]
+		fmt.Printf("Display name : %s", displayName)
+		fmt.Printf("Display name : %s", uuid)
```

Comment:
Dup text in printf "Display name"

Reply:
Removing these debug statements. Good catch